aten/src/ATen/core/jit_type.h

```diff
@@ -1555,6 +1555,8 @@ inline TypePtr TensorType::fromNumberType(TypePtr typ) {
     return TensorType::createContiguous(at::kDouble, at::kCPU, {});
   } else if (typ->isSubtypeOf(BoolType::get())) {
     return TensorType::createContiguous(at::kBool, at::kCPU, {});
+  } else if (typ->kind() == NumberType::Kind) {
+    return TensorType::create(c10::nullopt, at::kCPU, {}, c10::nullopt);
```

Comment:
Is there a way to add a test to demonstrate this behavior?

Reply:
this is created in the IR graph with `tensor.item()`